.github/workflows/cd.yaml

```diff
@@ -1,43 +1,99 @@
-name: CD
+name: Nightly CD
 
 on:
-  release:
-    types: [published]
+  push:
+    branches:
+      - master
 
 jobs:
-  appimage_amd64:
+  ubuntu_x86_64:
     runs-on: ubuntu-latest
     env:
-      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      ARCH: x86_64
-      OUTPUT: endless-sky-x86_64.AppImage
+      OUTPUT: endless-sky-amd64.tar.gz
```

Comment:
Is it intended that the downloaded filename never changes?

Reply:
It's a relic from when i tried to make this pipeline run on release and commit. I'll put the commit message in there (and give the AppImage a version while i'm at it).